client/state/data-layer/wpcom/comments/test/index.js

```diff
@@ -97,6 +99,74 @@ describe( 'wpcom-api', () => {
 			} );
 		} );
 
+		describe( '#writePostComment()', () => {
+			const action = {
+				type: 'DUMMY',
+				siteId: 101010,
```

Comment:
We usually point at `2916284` (the WordPress example blog) or `77203074` (another test site) to avoid pointing at a real user's site.